cpp/benchmarks/common/generate_nullmask.cu

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "generate_input.hpp"
+
+#include <cudf/detail/valid_if.cuh>
+#include <cudf/null_mask.hpp>
+
+#include <thrust/random.h>
+
+/**
+ * @brief bool generator with given probability [0.0 - 1.0] of returning true.
+ *
+ */
+struct bool_generator {
+  thrust::minstd_rand engine;
+  thrust::uniform_real_distribution<float> dist;
+  float probability_true;
+  bool_generator(thrust::minstd_rand engine, float probability_true)
+    : engine(engine), dist{0, 1}, probability_true{probability_true}
+  {
+  }
```

Comment:
Is this constructor ever used? I didn't see it being called. I would remove it if it's not being used.

Reply:
It is used in PR #10109

Reply:
```suggestion
```